content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -0,0 +1,1682 @@
+
+![TonyMcMapface](tm_tonymcmapface.png)
+
+### Blender Filmic
+
+From the 3D software package we know and love!
+
+![Blender Filmic](tm_blender_filmic.png)
+
+## Color Grading Control
+
+<div class="release-feature-authors">authors: @DGriffin91</div>
+
+We've added some basic control over color grading parameters such as exposure, gamma, "pre-tonemapping saturation", and "post-tonemapping saturation". These can be configured per-camera using the new [`ColorGrading`] component.
+
+[`ColorGrading`]: https://docs.rs/bevy/0.10.0/bevy/render/view/struct.ColorGrading.html
+
+### 0.5 Exposure
+
+![0.5 exposure](exposure_005.png)
+
+### 2.25 Exposure
+
+![2.25 exposure](exposure_225.png)
+
+## Parallel Pipelined Rendering
+
+<div class="release-feature-authors">authors: @hymm, @james7132</div>
+
+![Trace with Pipelined Rendering](pipelined-rendering-trace.png)
+
+On multithreaded platforms, **Bevy 0.10** will now run significantly faster by running simulation and
+rendering in parallel. The renderer was rearchitected in [Bevy 0.6](https://bevyengine.org/news/bevy-0-6/#pipelined-rendering-extract-prepare-queue-render)
+to enable this, but the final step of actually running them in parallel was not done until now.
+There was a bit of tricky work to figure out. The render world has a system that has to run on
+the main thread, but the task pool only had the ability to run on the world's thread. So, when we send
+the render world to another thread we need to accommodate still running render systems on the main
+thread. So we added the ability to spawn tasks onto the main thread in addition to the world's thread.
+
+![Histogram of Many Foxes Frame Time](pipelined-rendering-histogram.png)
+
+In testing different Bevy examples, the gains were typically in the 10% to 30% range.
+As seen in the above histogram, the mean frame time of the "many foxes" stress test
+is 1.8ms faster than before.
+
+To use pipelined rendering, you just need to add the `PipelinedRenderingPlugin`. If you're
+using `DefaultPlugins` then it will automatically be added for you on all platforms except
+wasm. Bevy does not currently support multithreading on wasm which is needed for this
+feature to work. If you are not using `DefaultPlugins` you can add the plugin manually.
+
+## Windows as Entities
+
+<div class="release-feature-authors">authors: @aceeri, @Weibye, @cart</div>
+
+In previous versions of Bevy [`Window`] was represented as an ECS resource (contained in the `Windows` resource). In **Bevy 0.10** [`Window`] is now a component (and therefore windows are represented as entities).
+
+This accomplishes a number of goals:
+
+* It opens the doors to representing Windows in Bevy's scene system
+* It exposes Windows to Bevy's powerful ECS queries
```

Comment:
using a capital W and no inline code here makes it seem a bit like talking about the OS and not like the actual window entities.

Reply:
```suggestion
* It exposes `Windows` to Bevy's powerful ECS queries
```